Utilities/bootstrap

```diff
@@ -1386,6 +1386,7 @@ def main():
         # Copy the libSwiftPM modules.
         # FIXME: This shouldn't require so much knowledge of the manifest.
         # FIXME: We should handle what happens if it's a static library.
+        src_path = os.path.join(target_path, conf, libswiftpm_product.product_name)
```

Comment:
Is this required while copying the C target? Seems like we need dst_path and not src_path in that case

Reply:
I mean we can avoid using it at all and just make this change for the `target.is_c` branch:

```
note("copying %s: %s" % (os.path.basename(src_path), ' '.join(cmd)))
```

to 
```
note("copying to %s: %s" % (dst_path, ' '.join(cmd)))
```